cmd/deploy/start.go

```diff
@@ -94,6 +107,7 @@ func NewDeployStartCmd(configuration *config.Configuration) *cobra.Command {
 			return start(cmd, configuration, options)
 		},
 	}
+	cmd.Flags().StringVarP(&options.account, "account", "", "", "override the deployment YAML account field for each target when --file is a URL")
```

Comment:
Does this allow for a short flag (`-a`)?

Reply:
We already have a `-a` unfortunately. It's used for passing in an auth token and used by the Aurora plugin. <-- look that came up twice in one week!